src/game/Objects/Unit.cpp

```diff
@@ -11425,6 +11425,17 @@ void Unit::TransferAttackersThreatTo(Unit* unit)
     Cell::VisitGridObjects(this, worker, 50.0f);
 }
 
+void Unit::RemoveAttackersThreat(Unit* owner)
+{
+    Unit::AttackerSet attackers = getAttackers();

```

Comment:
Shouldn't this be a reference, instead of copy?

Reply:
Yeah that was unecessary removed it.